### PR TITLE
Maxf/display inspire

### PR DESCRIPTION
**Pull request by maxf**

### CODE REVIEW

File: app/views/datasets/_additional_info.html.erb [hunk 1]
```diff
@@ -1,9 +1,89 @@
...
-  </p>
-</section>
+<% if @dataset.notes.present? || @dataset.inspire_dataset.present? %>
+
+  <section class="dgu-additional-info">
+    <h2 class="heading-medium">
+      <%= t('.additional_info') %>
+    </h2>
+
+    <% if @dataset.notes.present? %>
```
Comment (laurentqro) [inline app/views/datasets/_additional_info.html.erb]: Checking for the presence of something before displaying is such a common pattern in Rails that it provides us with `presence`, which is exactly equivalent to `object.present? ? object : nil`.

https://apidock.com/rails/Object/presence

Comment (maxf) [reply app/views/datasets/_additional_info.html.erb]: I'm not sure how you'd be using it in a template. Can you give an example?

Comment (laurentqro) [reply app/views/datasets/_additional_info.html.erb]: Just noticed notes are isnide a `<p>` tag, please ignore my comment, it's not relevant here.

**Comment by laurentqro** [comment]
Silly question; what's "inspire"?

**Pull request merged**